<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffOpenDialog</class>
 <widget class="QDialog" name="DiffOpenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select files/folders</string>
  </property>
  <property name="windowIcon">
   <iconset theme="kommit"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonCompareFiles">
     <property name="text">
      <string>Compare files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonCompareFolders">
     <property name="text">
      <string>Compare directories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetSelectFiles" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Old file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="UrlRequester" name="lineEditOldFile" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>New file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="UrlRequester" name="lineEditNewFile" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetSelectDirectories" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>New directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Old directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="UrlRequester" name="lineEditOldDirectory" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="UrlRequester" name="lineEditNewDirectory" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlRequester</class>
   <extends>QWidget</extends>
   <header>widgets/urlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiffOpenDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiffOpenDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCompareFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetSelectFiles</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCompareFolders</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetSelectDirectories</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
